benchmarks/README.md

```diff
@@ -88,7 +88,7 @@ cd inference-server/text-generation-inference
 # Copy the sample variables and update the project number and cluster name in
 # the fleet_host variable "https://connectgateway.googleapis.com/v1/projects/<project-number>/locations/global/gkeMemberships/<cluster-name>"
 # in the `terraform.tfvars` file.
-cp ./sample-tfvars/gpu-sample.tfvars terraform.tfvars
+cp ./sample-terraform.tfvars terraform.tfvars
```

Comment:
I dont see a corresponding changing in pathing, odd merge perhaps?

Reply:
correct, this is a bug fix because tgi is a GPU only workload, so it doesn't have multiple tfvars

ref: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/benchmarks/inference-server/text-generation-inference/sample-terraform.tfvars